src/Extensions/Services.Remoting/Internal/Diagnostics.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.Services.Remoting
+{
+	internal static class Diagnostics
+	{
+		internal const string ListenerName = "Microsoft.Omex.Extensions.Services.Remoting";
```

Comment:
Is there a way to get the name of the namespace here instead of hard coding, like nameof(class)?

Reply:
we could probably do something with reflection, but the downside would be that consumers may listening exactly this name and seemly harmless renaming of assembly disable telemetry from it,  I should probably add some comments about it

Reply:
Sure, I was wondering only if there is some simple nameof() equivalent here. Please ignore.

Reply:
Added comment and unit tests to check that name correspond assembly